### ISSUE TITLE
Genome assembly update: Drosophila biarmipes

### ISSUE THREAD

**Issue by mpoelchau** [NAL-i5K/general_issues#207]
This is a fairly straightforward genome update. Somehow I missed this in the last round of Drosophila updates. 

# Change old content (@mpoelchau )
- [ ] back up and deprecate old Apollo2
- [ ] back up and deprecate old fasta files in BLAST 
- [ ] back up and deprecate old fasta files in Hmmer 

# Add new content to i5k Workspace applications
See https://gitlab.com/i5k_Workspace/workspace_roadmap/-/wikis/Adding-an-organism-CWL-update for full description of each task (requires gitlab login)

- assembly accession number: GCF_018148935.1 
- publication associated with assembly: https://doi.org/10.7554/eLife.66405
- due date: 11/8/2022
- community contact: i5k Workspace

## Tasks on Ceres:
- no functional annotations needed - these are already done
- [x] 2. Run the final_workflow.cwl component of the organism_onboarding pipeline (https://gitlab.com/i5k_Workspace/workspace_roadmap/-/wikis/Adding-an-organism-CWL-update#1-run-the-final_workflowcwl-component-of-the-organism_onboarding-pipeline-httpsgithubcomnal-i5korganism_onboarding)
- [x] 3. Run RNA-Seq pipeline, steps 1 and 2 (https://gitlab.com/i5k_Workspace/workspace_roadmap/-/wikis/Adding-RNA-Seq-data-to-an-i5k-Workspace-project-with-the-NAL_RNA_seq_annotation_pipeline)

## Tasks on the NAL servers:  (@amcooksey you'll need to wait for me to have cleaned up the servers first before starting on this list)
- [x] 1. Move the final_workflow.cwl output from SCINet to apollo2-stage-node1 (https://gitlab.com/i5k_Workspace/workspace_roadmap/-/wikis/Adding-an-organism-CWL-update#3-move-the-final_workflowcwl-output-from-scinet-to-apollo2-stage-node1)
- [x] 2. Apollo setup on the apollo2-stage-node1 and apollo2-node1 servers (https://gitlab.com/i5k_Workspace/workspace_roadmap/-/wikis/Adding-an-organism-CWL-update#4-apollo-setup-on-the-apollo2-stage-node1-and-apollo2-node1-servers) 
- [x] 3. Set up Tripal pages (https://gitlab.com/i5k_Workspace/workspace_roadmap/-/wikis/Adding-an-organism-CWL-update#6-set-up-tripal-pages)
- [x] 4. Set up BLAST and HMMER (genomics-workspace) (https://gitlab.com/i5k_Workspace/workspace_roadmap/-/wikis/Adding-an-organism-CWL-update#7-set-up-blast-and-hmmer-genomics-workspace)
- [x] 5. Set up data downloads (https://gitlab.com/i5k_Workspace/workspace_roadmap/-/wikis/Adding-an-organism-CWL-update#8-set-up-data-downloads)
- [ ] 6. Final Test List (https://gitlab.com/i5k_Workspace/workspace_roadmap/-/wikis/Adding-an-organism-CWL-update#10-final-test-list)
- [ ] 7. Advertising (https://gitlab.com/i5k_Workspace/workspace_roadmap/-/wikis/Adding-an-organism-CWL-update#10-advertising) (@mpoelchau)

**Comment by mpoelchau** [comment]
@amcooksey this okay to go ahead with on stage - you can create a new directory in other_species, the old one is in modencode
